packages/bpk-component-select/src/bpk-select.scss

```diff
@@ -41,3 +41,48 @@
     }
   }
 }
+
+@mixin bpk-select-changeme {
+  display: flex;
+  padding-right: 0;
+  flex-direction: row;
+  align-items: center;
+  background: none;
+}
+
+.bpk-select-changeme {
```

Comment:
Need a better name for this, ideas?

Reply:
It sounds like this should be a modifier, eg `bpk-select--with-image`. You wouldn't need to redefine all the other modifiers in this case. e.g. `className={getClassName('bpk-select', 'bpk-select--with-image')}`